src/environments/environment.ts

```diff
@@ -4,7 +4,16 @@
 
```

Comment:
Consider adding a placeholder firebaseConfig object in `environment.prod.ts`. Here we can later put prod specific values.

Reply:
Added firebase config details placeholder in ` environment.prod.ts`
[Commit link](https://github.com/armory-uci/vital/pull/8/commits/066d6038cd7ea68ebd6bcc242bd6fed17dc374fb)